File: 1630-count-odd-numbers-in-an-interval-range/count-odd-numbers-in-an-interval-range.py
import math

class Solution:
    def countOdds(self, low: int, high: int) -> int:
        dif = high-low 
        count = math.floor(dif/2)
        if low % 2 == 1 or high % 2 == 1:
            count += 1
        # if high % 2 == 1:
        #     count += 1
        return count

File: 1630-count-odd-numbers-in-an-interval-range/count-odd-numbers-in-an-interval-range.cpp
class Solution {
public:
    int countOdds(int low, int high) {
        int count = floor((high-low)/2);
        // If low is even, make it odd.
        if ((low & 1) || (high & 1)) {
            count++;
        }
  
        // low could become greater than high due to incrementation
        // if it is, the answer is 0; otherwise, use the formula.
        return count;
    };
};